app/controllers/destinations_controller.rb

```diff
@@ -3,10 +3,16 @@ def index
         destinations = Destination.all
         render json: destinations
     end
+
+    def update
+        destination = Destination.find(params[:id])
+        destination.update(destination_params)
+        if destination.valid?
+            render json: destination
+        else
```

Comment:
I don't see anything after the else here

Reply:
We have runaway code!

Reply:
🤣🏃‍♀️